### PR TITLE
make noarch

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

One more for https://github.com/conda-forge/conda-forge.github.io/issues/1840

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,22 +12,24 @@ source:
...
   run:
+    - __{{ target_os }}
     - conda >=4.6
-    - python
+    - python >=3.6
     - ruamel_yaml >=0.11.14,<0.16
     - conda-standalone
     - pillow >=3.1  # [win]
-    - nsis >=3.01  # [win]
+    - nsis >=3.01,<3.08  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:32]: Why is the upper bound here needed?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,22 +12,24 @@ source:
...
   entry_points:
     - constructor = constructor.main:main
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
+    - __{{ target_os }}
```
Comment (jakirkham) [inline recipe/meta.yaml:26]: Nice! 😄

**Review by jakirkham — COMMENTED** [review]
Thanks Jaime! 🙏

LGTM. Had a question below

File: recipe/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipe/meta.yaml:32]: Mitigation for https://github.com/conda/constructor/issues/526

We patched it in repodatas too: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/300

**Comment by jakirkham** [comment]
Thanks Jaime! 🙏

**Pull request merged**